docs/dockermachine.md

```diff
@@ -0,0 +1,449 @@
+page_keywords: docker, machine, virtualbox, digitalocean, amazonec2
+ 
+# Working with Docker Machine
+
+## Overview
+
+In order to run Docker containers, you must have a 
+[Docker daemon](https://docs.docker.com/arch) running somewhere. If you’re on a 
+Linux system and you want to run a container on your local machine, this is 
+straightforward: you run the daemon on your local machine and communicate with 
+it over the Unix socket located at `/var/run/docker.sock` (this all happens 
+behind the scenes when you run `docker` on the command line). However, if you 
+want to control containers from Mac OSX / Windows or manage them on a remote 
+server, you’ll need to create a new machine (probably a virtual machine) with 
+Docker installed and execute Docker commands for that host remotely.  
+Traditionally, the way to do this was either:
+
+- manual (open the web interface or virtualization application, make the machine 
+yourself, manually install Docker, etc.) and therefore tedious and error-prone
+- with existing automation technologies, which usually entail a quite high skill 
+threshold
+
+Docker's [`machine`](https://github.com/docker/machine) is a tool for making the 
+process of creating and managing those machines (and running Docker commands 
+against them) much faster and easier for users.  `machine` allows users to 
+quickly create running instances of the Docker daemon on local virtualization 
+platforms (e.g. Virtualbox) or on cloud providers (e.g. AWS EC2) that they can 
+connect to and control from their local Docker client binary.
+
+## Installation
+
+Docker Machine is supported on Windows, OSX, and Linux.  To install Docker 
+Machine, download the appropriate binary for your OS and architecture to the 
+correct place in your `PATH`:
+
+- [Windows - x86_64]()
+- [OSX - x86_64]()
+- [Linux - x86_64]()
+- [Windows - i386]()
+- [OSX - i386]()
+- [Linux - i386]()
+
+Now you should be able to check the version with `machine -v`:
+
+```
+$ machine -v
+machine version 0.0.2
+```
+
+## Getting started with Docker Machine using a local VM
+
+Let's take a look at using `machine` to creating, using, and managing a Docker 
+host inside of [VirtualBox](ihttps://www.virtualbox.org/).
+
+First, ensure that 
+[VirtualBox 4.3.20](https://www.virtualbox.org/wiki/Downloads) is correctly 
+installed on your system. 
+
+If you run the `machine ls` command to show all available machines, you will see 
+that none have been created so far.
+
+```
+$ machine ls
+NAME   ACTIVE   DRIVER   STATE   URL
+```
+
+To create one, we run the `machine create` command, passing the string 
+`virtualbox` to the `--driver` flag.  The final argument we pass is the name of 
+the machine - in this case, we will name our machine "dev".
+
+This will download a lightweight Linux distribution 
+([boot2docker](https://github.com/boot2docker/boot2docker)) with the Docker 
+daemon installed, and will create and start a VirtualBox VM with Docker running.
+
+
```

Comment:
> so we need a new docker binary first :/

Well, the doc was originally written assuming that by time of publication the trust PR would be in docker main (1.5?).  Bold assumption, I know, so maybe we _should_ add a section on the custom Docker binary and remove it when that PR gets in.

As far as not mixing well with existing TLS files, that seems like it would merit its own separate issue with steps to reproduce (and consequently make `machine` code more robust to deal with it), although I agree documentation should cover that possibility, at least in a "Troubleshooting" section.